config/routes.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/graphql"
+  post "/graphql", to: "graphql#execute"
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
```

Comment:
Can we also add documentation with the different types/endpoints as we [already do in Decidim](https://www.decidim.barcelona/api/docs) :pray: ?